packages/thirdweb/src/bridge/index.ts

```diff
@@ -0,0 +1,8 @@
+export * as Buy from "./Buy.js";
+export * as Sell from "./Sell.js";
```

Comment:
you might need to add these as entrypoints in scripts/typedoc.mjs so its picked up correctly by the autogen docs

Reply:
reminder to add this